boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -100,17 +99,18 @@
 };
 
 &lpi2c2 {
-	status = "okay";
+	status = "disabled";
```

Comment:
why still disable this?

Reply:
we can have this enabled by default. updated.